jenkins/ci.suse.de/cloud-mediacheck.yaml

```diff
@@ -40,6 +40,15 @@
         (["Devel:Cloud:5", "Devel:Cloud:5/SLE_12", "Devel:Cloud:6", "Devel:Cloud:6/SLE_12_SP1", "Devel:Cloud:7/SLE_12_SP2" ].contains(project) || subproject == ":")
     builders:
       - shell: |
+          export jtsync=${automationrepo}/scripts/jtsync/jtsync.rb
+          function jtsync_trap() {
+            $jtsync --ci suse --matrix ${JOB_BASE_NAME},${project},${BUILD_NUMBER} 1
+          }
+
+          # only enable jtsync when build is not manually triggered
+          if [ "x${ROOT_BUILD_CAUSE}" != "xMANUALTRIGGER" ]; then
```

Comment:
save the 'x' and quotes using double brackets:
`if [[ $ROOT_BUILD_CAUSE != "MANUALTRIGGER" ]]; then`